addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -129,8 +129,8 @@ export class DiscussCoreCommon {
             channel.markAsRead();
         }
         this.env.bus.trigger("discuss.channel/new_message", { channel, message, silent });
-        const authorMember = channel.channel_member_ids.find(({ persona }) =>
-            persona?.eq(message.author)
+        const authorMember = channel.channel_member_ids.find(
+            (m) => m.partner_id?.eq(message.author_id) || m.guest_id?.eq(message.author_guest_id)
```

Comment:
This is an example of one of those cases where I don't really see the value of fully removing the getter.